### ISSUE TITLE
USWDS - Feature: Dynamic tooltip changes

### ISSUE THREAD

**Issue by sarahwylie** [uswds/uswds#6426]
### Is your feature request related to a problem? Please describe.

The tooltip can be created dynamically, but it currently cannot be updated once created. If the state of a button changes, for example, the tooltip text on that button cannot update to reflect the new state change. Instead, multiple tooltips are created and continue to propagate. 

### Describe the solution you'd like

In the `setUpAttributes()` function, wrap a conditional around the creation of the `span` element so additional tooltips are not built on top of the original.

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Feature+Request%22) for duplicate feature requests.

**Comment by venkatajanapareddy** [comment]
I've created a [CodeSandbox](https://codesandbox.io/p/sandbox/dynamic-tooltip-issue-fhqwfl) that reproduces this issue.

I was able to reproduce this problem in Google Chrome, Firefox, and Safari on a MacBook. Additionally, I noticed an extra issue where newly created tooltips persist in the UI even after the mouse hover ends. Please follow the steps to reproduce mentioned in the CodeSandbox link to see this in action. This behavior can lead to screen readers announcing multiple, conflicting tooltips, and it causes the DOM to become unnecessarily cluttered.
